docs/source/api.rst

```diff
@@ -13,12 +13,11 @@ forward compatibility and will be unprefixed and converted to strings as
 discussed in :doc:`features`.
 
 
-ElasticSearch API
-=================
+ElasticSearch Class
```

Comment:
Nit mcnitty, but they changed it from ElasticSearch to Elasticsearch. I went through ElasticUtils and changed all instances. Might make sense for you to, too, at some point.